Header header

float32 encoder_step1
float32 encoder_step2
float32 encoder3
float32 encoder4


uint8  step1_finish_flag
uint8  step2_finish_flag
